modules/core/js/src/main/scala/com/azavea/stac4s/geometry/Geometry.scala

```diff
@@ -16,20 +16,20 @@ object Geometry {
   }
 
   case class Polygon private (coords: List[Point2d]) extends Geometry {
-    def asCoordinateArray: List[List[Double]] = coords.map(_.asCoordinateArray)
+    def asCoordinateArray: List[List[List[Double]]] = List(coords.map(_.asCoordinateArray))
   }
 
   case class MultiPolygon private (polys: List[Polygon]) extends Geometry {
-    def asCoordinateArray: List[List[List[Double]]] = polys.map(_.asCoordinateArray)
+    def asCoordinateArray: List[List[List[List[Double]]]] = polys.map(_.asCoordinateArray)
   }
```

Comment:
Just as a thing that was discussed: because of possible rings `Polygon` coordinates are stored as `List[List[List[Double]]]` and `Multiplogyons` coordinates as `List[List[List[List[Double]]]]`. That was revelaed during the JS Client integration testing.